src/module_wrap.cc

```diff
@@ -279,6 +330,14 @@ void ModuleWrap::New(const FunctionCallbackInfo<Value>& args) {
     return;
   }
 
+  if (!that->Set(context,
```

Comment:
I think it would be better to invoke `getModuleRequestsSync()` in the ModuleJob constructor, because not all constructions of ModuleWrap need this (e.g. if it is only used to check syntax).